rocAL_pybind/examples/rocAL_api_numpy_reader.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import random
+
+from amd.rocal.pipeline import Pipeline
+from amd.rocal.plugin.pytorch import ROCALNumpyIterator
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import sys
+import os
+
+def main():
+    if  len(sys.argv) < 3:
```

Comment:
It should be 'if len(sys.argv) < 4' here, as you require three arguments, and argv[0] by default contains the Python script name.